src/main/java/org/apache/sysds/runtime/instructions/cp/MultiReturnBuiltinCPInstruction.java

```diff
@@ -47,6 +47,10 @@ public CPOperand getOutput(int i) {
 		return _outputs.get(i);
 	}
 
+	public ArrayList<CPOperand> getOutputs(){
```

Comment:
Maybe change these to List type return?

Reply:
I also thought about doing that. It will always be an ArrayList, so in the end it would not make much of a difference. It is more an API-design thing. Do you prefer the List type?

Reply:
well, its an opinion thing, i like to return the highest abstraction, i think we had a course this was mentioned as a point.
Software Engineering... :+1:

Reply:
Yes, I usually also go for that option.

Reply:
I have changed it now.